### DISCUSSION TITLE
Dark mode/theme?

### DISCUSSION THREAD

**Discussion by xMdb** [firefly-iii/firefly-iii#5629] [Firefly III]
Is/when will there be a native dark mode/theme?

**Comment by JC5** [comment]
There isn't one, it's not featured in the layout I'm using. The new layout I'm working on has it, but there's no ETA yet.

**Comment by thorrrr** [comment]
HI 
Great work very impressive but it really needs a Dark Mode it a real retina burner.
Have you made any progress on the Dark Mode or do you have a beta version to try out?

**Reply by supermamon** [reply]
I just tried the beta version has dark mode that follows the system setting. I personally would like if there's an option not to follow the system setting and either always be in dark or light mode.

https://github.com/firefly-iii/firefly-iii/releases/tag/v6.0.0-beta.1

**Reply by AlexKalopsia** [reply]
Can't agree more. I appreciate the effort with the dark theme support, but it's not very readable and I'd rather be able to set the theme manually.
